src/utils/spec/file.spec.ts

```diff
@@ -41,4 +42,19 @@ describe('file utility', () => {
       deleteFolder(path.join(process.cwd(), 'testFolder_1'))
     })
   })
+
+  describe('unifyFilePath', () => {
+    it('should unify file path in Unix-like systems', () => {
```

Comment:
Do you think it's worth adding a test case for when the separators are mixed up?

Reply:
good point!
updated `unifyFilePath` utility